packages/databyss-ui/components/PageContent/PageContent.js

```diff
@@ -69,31 +63,34 @@ const PageContainer = React.memo(({ anchor, id, page }) => {
     }
   }
 
+  const onEditorChange = editorStateRef => {
+    setEditorState(editorStateRef?.state)
+  }
+
   return (
-    <View height="100vh" overflow="scroll" p="medium">
-      <View
-        mr="medium"
-        alignItems="center"
-        flexDirection="row"
-        justifyContent="space-between"
-      >
-        <PageHeader
-          ref={headerRef}
-          pageId={id}
-          onNavigateDownFromHeader={onNavigateDownToEditor}
+    <View height="100vh" overflow="scroll" ref={editorWindowRef}>
+      <PageSticky page={editorState} />
```

Comment:
I believe we discussed this, but let's make this prop just the list of "rendered" breadcrumbs in the path. This way, we can keep the logic that requires the editor state within the EditorProvider where it can access it directly, and just "publish" the rendered path elements on the ref. I think this is preferable to passing the whole state around.

Reply:
I just moved the `pagePath` logic to `EditorProvider`. It now passes a `string` instead of the whole state. I'm not sure what you mean by `rendered`.